sale_coupon_auto_refresh/models/sale_order_line.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2021 Tecnativa - David Vidal
+# Copyright 2021 Camptocamp - Silvio Gregorini
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class SaleOrderLine(models.Model):
+    _name = "sale.order.line"
```

Comment:
Why tearing this apart in another file. It's a not written (or maybe it is, I didn't checked :sweat_smile:) convention to have o2m relations attached to its father in the same file. Anyway, for this case it would ease the diff reading keeping it as it was.

Reply:
I checked the guidelines, which say that the rule is "one model per file" and having multiple models in the same file can be an exception (and it's not mandatory): 

> For models, views and data declarations, split files by the model involved, either created or inherited. [...] An exception can be made when the model is a model intended to be used only as a one2many model nested on the main model.
https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#file-naming

I know it's harder to diff files in this PR because of the split, but I found out it was easier to work this way instead of always having to scroll up and down to check if the code was ok on both models. I believe it'll be better this way on the long run.

Reply:
@SilvioC2C although the guidelines say that is optional to not split, the other guideline is to do the minimal diff and respect the style of previous work, so although you can init a new module and decide to split `sale.order` and `sale.order.line` in 2 files, in this module, that they are already merged, please don't add noise doing such split.

Reply:
@pedrobaeza thanks for the heads-up, I will remove the split then!